typescript/sdk/src/providers/transactions/transformer/ethersV5/EV5InterchainAccountTxTransformer.ts

```diff
@@ -37,18 +36,16 @@ export class EV5InterchainAccountTxTransformer
 
   public async transform(
     ...txs: AnnotatedEV5Transaction[]
-  ): Promise<ethers.PopulatedTransaction[]> {
+  ): Promise<AnnotatedEV5Transaction[]> {
     const txChainsToInnerCalls: Record<ChainName, CallData[]> = txs.reduce(
       (
         txChainToInnerCalls: Record<ChainName, CallData[]>,
-        { to, data, chainId }: AnnotatedEV5Transaction,
+        { to, data, chain }: AnnotatedEV5Transaction,
       ) => {
-        assert(chainId, 'Invalid PopulatedTransaction: chainId is required');
-        assert(to, 'Invalid PopulatedTransaction: to is required');
-        assert(data, 'Invalid PopulatedTransaction: data is required');
-        const txChain = this.multiProvider.getChainName(chainId);
-        txChainToInnerCalls[txChain] ||= [];
-        txChainToInnerCalls[txChain].push({ to, data });
+        assert(to, 'Invalid AnnotatedEV5Transaction: "to" is required');
+        assert(data, 'Invalid AnnotatedEV5Transaction: "data" is required');
```

Comment:
does this need an assert on `chain`?

Reply:
We don't *need* one because `AnnotatedEV5Transaction` extends from an ethers `PopulatedTransaction` where all fields are optional - so we need the assert to ensure values from that type are properly set.

```ts
import type { PopulatedTransaction as EV5Transaction } from 'ethers';

export type AnnotatedEV5Transaction = Annotated<EV5Transaction>;
```
<img width="318" alt="image" src="https://github.com/user-attachments/assets/08ae3664-8b24-4e1a-a760-d06f8a3239b5">


Whereas `chain` is a required property of the `Annotated<>` wrapper we're using


```ts
export type Annotated<T> = T & {
  chain: string;
  annotation?: string;
};
```

Reply:
makes sense!

Reply:
`Pick` and `Required` might be useful here
https://www.typescriptlang.org/docs/handbook/utility-types.html#requiredtype